htdocs/stripe/class/stripe.class.php

```diff
@@ -373,7 +373,7 @@ public function createPaymentStripe($amount, $currency, $origin, $item, $source,
 		);
 		$return = new Stripe($this->db);
 		try {
-			if (empty($conf->stripeconnect->enabled))
+			if (empty($conf->stripeconnect->enabled) || $conf->entity==1)
```

Comment:
Entity defined on wich company we are working.
Code and behaviour must be same whatever is entity. It can depends on setup that differs according to entity but not on the fact we are on an entity or another

Reply:
The problème is that with stripe connect, code must be different for a principal entity (as a non connect) or a connect entity... Behavior of a principal entity with connect is the same as a non connect entity, which code is different of a entity in connect mode... Principal entity should be the first as a principal multicompany